kafka-connect-cloud-common/src/main/scala/io/lenses/streamreactor/connect/cloud/common/formats/reader/converters/SchemaAndValueEnvelopeConverter.scala

```diff
@@ -80,13 +81,15 @@ class SchemaAndValueEnvelopeConverter(
     if (fields.contains("key")) {
       key       = struct.get("key")
       keySchema = struct.schema().field("key").schema()
+      key       = byteBufferToArray(keySchema, key)
     }
 
     var value:       Any    = null
     var valueSchema: Schema = null
     if (fields.contains("value")) {
       value       = struct.get("value")
       valueSchema = struct.schema().field("value").schema()
+      value       = byteBufferToArray(valueSchema, value)
     }
```

Comment:
Same comment as above but will raise you a "this code is identical*, can it be refactored into a reusable function"?

* with the exception of some variable names